### PR TITLE
Format python in docstrings

### PR DESCRIPTION
**Pull request by jarrodmillman**
@dschult @rossbar This started as an attempt to run black on the docstrings.  When I tried to do that, I noticed that several files no longer were formatted according to black (fb9bea4).  Then I ran black on the docstrings and incorrectly converted a couple docstrings to f-strings (dfe958d).   I also did some manual cleanup and then quieted some pytest warnings.  I looked at everything fairly closely and am reasonably confident this didn't mess anything up.  I would like to get this merged soon, so let me know if this looks OK.

We should set up the CI system to automatically run black on the code (and maybe on the docstrings) on each PR soon.

### CODE REVIEW

File: networkx/algorithms/tests/test_max_weight_clique.py [hunk 1]
```diff
@@ -10,83 +10,259 @@ class TestMaximumWeightClique:
...
-             (15, 28), (16, 17), (16, 18), (16, 19), (16, 20), (16, 21),
-             (16, 29), (16, 30), (17, 18), (17, 21), (17, 22), (17, 25),
-             (17, 27), (17, 28), (17, 30), (18, 19), (18, 20), (18, 21),
-             (18, 22), (18, 23), (18, 24), (19, 20), (19, 22), (19, 23),
-             (19, 24), (19, 25), (19, 27), (19, 30), (20, 21), (20, 23),
-             (20, 24), (20, 26), (20, 28), (20, 29), (21, 23), (21, 26),
-             (21, 27), (21, 29), (22, 24), (22, 25), (22, 26), (22, 29),
-             (23, 25), (23, 30), (24, 25), (24, 26), (25, 27), (25, 29),
-             (26, 27), (26, 28), (26, 30), (28, 29), (29, 30)]
+            [
```
Comment (rossbar) [inline networkx/algorithms/tests/test_max_weight_clique.py:50]: I might quibble with this one - it goes from ~30 -> ~200 LOC and arguably makes the test more difficult to read. I'd vote for practicality to override style in this case but not a blocker

File: networkx/tests/test_relabel.py [hunk 2]
```diff
@@ -184,8 +184,8 @@ def test_relabel_selfloop(self):
         G = nx.relabel_nodes(G, {1: 0}, copy=False)
         assert_nodes_equal(G.nodes(), [0])
 
-#    def test_relabel_multidigraph_inout_inplace(self):
-#        pass
+    #    def test_relabel_multidigraph_inout_inplace(self):
```
Comment (rossbar) [inline networkx/tests/test_relabel.py:187]: I don't know about this change either (though the commented test should probably just be removed)

Comment (jarrodmillman) [reply networkx/tests/test_relabel.py]: Removed.

File: networkx/algorithms/tests/test_max_weight_clique.py (hunk 1 above)
Comment (jarrodmillman) [reply networkx/algorithms/tests/test_max_weight_clique.py:51]: Used ``fmt: off`` and  ``fmt: on``.

**Pull request merged**